src/absil/il.fs

```diff
@@ -77,21 +81,21 @@ let rec splitNamespaceAux (nm:string) =
 
 /// Global State. All namespace splits ever seen
 // ++GLOBAL MUTABLE STATE
+#if FABLE_COMPILER
```

Comment:
Why not conditionally define 

    type ConcurrentDictionary<'K,'V> = Dictionary<'K'V>

?